cmds/ocm/commands/ocmcmds/common/addhdlrs/options.go

```diff
@@ -8,30 +8,51 @@ import (
 )
 
 type Options struct {
-	Replace bool
+	Replace           bool
+	PreserveSignature bool
 }
 
-var _ ocm.ModificationOption = (*Options)(nil)
+var (
+	_ ocm.ModificationOption        = (*Options)(nil)
+	_ ocm.ElementModificationOption = (*Options)(nil)
+	_ ocm.BlobModificationOption    = (*Options)(nil)
+	_ ocm.TargetElementOption       = (*Options)(nil)
+)
 
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	f := fs.Lookup("replace")
-	if f != nil {
-		if bp := generics.Cast[*bool](f.Value); bp != nil {
-			return
-		}
+	if f == nil {
+		fs.BoolVarP(&o.Replace, "replace", "R", false, "replace existing elements")
+	}
+
+	f = fs.Lookup("preserve-signature")
+	if f == nil {
+		fs.BoolVarP(&o.PreserveSignature, "preserve-signature", "P", false, "preserve existing signatures")
```

Comment:
```suggestion
		fs.BoolVarP(&o.PreserveSignature, "preserve-signature", "P", false, "preserve existing signatures")
```
similar to above. I think we should write here when to use this flag.

Reply:
there is a more detailed description as part of the command docu.